rules/falco_rules.yaml

```diff
@@ -2708,7 +2708,7 @@
   output: >
     Symlinks created over senstivie files (user=%user.name user_loginuid=%user.loginuid command=%proc.cmdline target=%evt.arg.target linkpath=%evt.arg.linkpath parent_process=%proc.pname)
   priority: NOTICE
-  tags: [file, mitre_exfiltration]
+  tags: [file, mitre_TA0010_exfiltration, mitre_TA0005_defense_evasion, mitre_T1036_masquerading, mitre_T1036.005_match_legitimate_name_or_location]
```

Comment:
not fit for `mitre_TA0005_defense_evasion, mitre_T1036_masquerading, mitre_T1036.005_match_legitimate_name_or_location`